web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -273,12 +280,20 @@ public String handleUvViewerRequest(@PathVariable("pid") String pidString, Model
     private Map<String, Object> getViewerProperties(ContentObjectRecord briefObject, AccessGroupSet principals) {
         String viewerType = null;
         String viewerPid = null;
+        String streamingUrl = null;
+        String streamingType = null;
 
         boolean imageViewerNeeded = accessCopiesService.hasViewableFiles(briefObject, principals);
+        var workStreamingContent = accessCopiesService.getStreamingChildren(briefObject, principals);
```

Comment:
We could probably refactor this method a bit so that we only execute this query when we need it, since at the moment it will get called for uv or streaming files unnecessarily. But it might be an unnecessary optimization.